MultithreadedProgram/ThreadBuilder.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Threading;
+
+namespace MultithreadedProgram
+{
+    public class ThreadBuilder
+    {
+        public static void BuildAThread()
+        {
+            Writer writer = new Writer(RandomHelper.GetRandomSymbol(), RandomHelper.GetRandomColor(), RandomHelper.GetRandomDelay());
```

Comment:
Please define the parameters to variables. It would be much cleaner and easier when debugging.